charts/rancher-vsphere-cpi/values.yaml

```diff
@@ -9,12 +9,21 @@ vCenter:
     name: "vsphere-cpi-creds"
     generate: true
 
-# vSphere Tags used to determine the zone and region of a Kubernetes node. This labels will be propagated to NodeLabels
+  # vSphere Tags used to determine the zone and region of a Kubernetes node. This labels will be propagated to NodeLabels
   labels:
     region: "k8s-region"
     zone: "k8s-zone"
     generate: false
 
+  ipFamily: []
```

Comment:
needs to be a string as noted. Please also add a comment describing usage.

Reply:
For `ipFamily` the docs are a little misleading in that they show the legacy ini config where it was a single string and not the [current yaml config](https://github.com/kubernetes/cloud-provider-vsphere/blob/a096bb88b449efd2bd980c8855393f3cf593b330/pkg/common/config/types_yaml.go#L71) where it is a string array. I've had to figure most of this out from the source rather than the docs.

Will add a comment.